src/commands.cc

```diff
@@ -529,6 +532,18 @@ const CommandDesc buffer_cmd = {
         {
             context.push_jump();
             context.change_buffer(buffer);
+
+            const size_t param_count = parser.positional_count();
+            if (param_count > 1 and not parser[1].empty())
```

Comment:
I am not sure we need to test for empty, str_to_int will return 0 in those cases. (Maybe it should fail with "not a number", in any case we dont gain much by testing here).